packages/components/src/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,143 @@
+import React, {
+  ForwardRefExoticComponent,
+  RefAttributes,
+  forwardRef,
+} from 'react';
+import {
+  Link,
+  Breadcrumbs as RACBreadcrumbs,
+  BreadcrumbsProps as RACBreadcrumbsProps,
+} from 'react-aria-components';
+import { cn, useClassNames } from '@marigold/system';
+import { ChevronRight } from '../icons';
+import { BreadcrumbEllipsis } from './BreadcrumbEllipsis';
+import { BreadcrumbsItem, BreadcrumbsItemProps } from './BreadcrumbsItem';
+
+type RemovedProps = 'className' | 'style' | 'children' | 'isDisabled';
+
+export interface BreadcrumbsProps
+  extends Omit<RACBreadcrumbsProps<object>, RemovedProps> {
+  variant?: string;
+  size?: string;
+
+  /**
+   * Disables the breadcrumbs.
+   * @default false
+   */
+  disabled?: RACBreadcrumbsProps<object>['isDisabled'];
+
+  /**
+   * Maximum number of visible items before the breadcrumbs collapse.
+   * @default 3
+   */
+  maxVisibleItems?: number;
+
+  /**
+   * Type of separator between breadcrumb items.
+   * @default 'chevron'
+   */
+  separatorType?: 'chevron' | 'slash';
```

Comment:
I think we can start without this and only start with a fixed separator. Otherwhise people will use what they like, so we need to decide for them :D